classes/Pods/Data.php

```diff
@@ -978,7 +978,20 @@ public function build( $params ) {
 		}
 
 		if ( ! empty( $params->orderby ) ) {
+			if ( 'post_type' == $pod[ 'type' ] && 'meta' == $pod[ 'storage' ] && is_array( $params->orderby ) ) {
+				foreach ( $params->orderby as $i => $orderby ) {
+					if ( strpos( $orderby, '.meta_value_num' ) ) {
+						$params->orderby[ $i ] = 'CAST(' . str_replace( '.meta_value_num', '.meta_value', $orderby ) . ' AS DECIMAL)';
+					} elseif ( strpos( $orderby, '.meta_value_date' ) ) {
+						$params->orderby[ $i ] = 'CAST(' . str_replace( '.meta_value_date', '.meta_value', $orderby  ) . ' AS DATE)';
+					}
+
+				}
+
+			}
+			
 			$params->orderby = (array) $params->orderby;
```

Comment:
Hmm thats, how I did it here: https://github.com/pods-framework/pods/commit/f6652895a1274c1305a170d67fec6c4f9593d508#diff-a96f9b1c49edf7ebf160a14d0b7b3f2aR901

$params->orderby is already an array by the time it gets there, at least in 2.x